launch/gazebo.launch

```diff
@@ -16,5 +16,5 @@
     <node ns="robotont" name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
     <node ns="robotont" name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
     <!-- Spawn a robot into Gazebo -->
-    <node name="spawn_urdf" pkg="gazebo_ros" type="spawn_model" args="-param /robotont/robot_description -urdf -x -$(arg x_pos) -model robotont" />  
+    <node ns="robotont" name="spawn_urdf" pkg="gazebo_ros" type="spawn_model" args="-param /robotont/robot_description -urdf -x -$(arg x_pos) -model robotont" />  

```

Comment:
ega siin ei toimu nüüd topelt namespace'imist? ma enam ei suuda ei roboti mudelit ega laserscani rvizis visualiseerida

Reply:
pidin rvizi robot_descriptioni muutma robotont/robot_description'iks, siis kadusid TF errorid

Reply:
ja scani ei õnnestu mul üldse visualiseerida. LaserScan plugin annab errorit: `For frame [robotont/hokuyo_link]: Frame [robotont/hokuyo_link] does not exist`

Reply:
oh pagan. siin on probleem pisut laiem. meil ei ole ühtne descriptioni parameetriserverisse laadimine. mõni launch laadib robotont/robot_description, teine jälle otse robot_descriptionisse.

ilmselt on lahendus luua description kimpudesse oma upload-launchid, mis tegelevad parameetriserverisse kirjelduse laadimisega ning ükski teine kimp ei tee omaloomingut, vaid kasutab neid uploadereid.

teen selle kohta eraldi issue